.github/workflows/installer.yml

```diff
@@ -0,0 +1,57 @@
+name: Installer integration
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "installer.sh"
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  ubuntu:
+    name: Test installer on Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Install code-server
+        run: ./install.sh
+
+      - name: Test code-server
+        run: yarn test:standalone-release code-server
+
+  alpine:
+    name: Test installer on Alpine
+    runs-on: ubuntu-latest
+    container: "alpine:3.14"
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+
+      - name: Install curl
+        run: apk add curl
+
+      - name: Add user
+        run: adduser coder --disabled-password
```

Comment:
Why do we have to add a user and why do we use `--disabled-password`?

Reply:
It's to make sure the script works without being ran as root (to make sure we don't have permission issues). It's basically to make sure https://github.com/cdr/code-server/issues/3585 doesn't happen. I linked that in the commit message but it might make sense to add as a comment.